cmd/ftl/cmd_dev.go

```diff
@@ -138,6 +138,7 @@ func (d *devCmd) Run(ctx context.Context, client ftlv1connect.ControllerServiceC
 				deploy := deployCmd{
 					Replicas:  1,
 					ModuleDir: dir,
+					Wait:      d.ExitAfterDeploy,
```

Comment:
if `ExitAfterDeploy` is true, we'll pass `Wait` for all the deploys contained therein